object_storage/tests/integrated.rs

```diff
@@ -0,0 +1,21 @@
+use fractal_object_storage::{test::InMemoryDb, FractalStore, Given, ProofChecker, Proposition};
+
+#[test]
+fn proving_simple_value() {
```

Comment:
Maybe we can add more integrated tests for cases like - multiple same proofs, removing proofs, etc.